src/App.tsx

```diff
@@ -24,13 +32,43 @@ const router = createBrowserRouter([
     path: "/employee",
     element: <EmployeePage />,
   },
+  {
+    path: "/admin/users",
+    element: <UsersOverview />,
+  },
+  {
+    path: "/admin/users/add",
+    element: <UserAdd />,
+  },
+  {
+    path: "/admin/users/:userId",
+    element: <EditUser />,
+  },
+  {
+    path: "/admin/destinations",
+    element: <DestinationsOverviewPage />,
+  },
+  {
+    path: "/admin/destinations/add",
+    element: <AddDestinationPage />,
+  },
+  {
+    path: "/admin/destinations/:destinationId",
+    element: <EditDestinationPage />,
+  },
+  {
+    path: "/page-not-found",
```

Comment:
Does this work as a 404 page? Are components redirecting to this by any chance?

Reply:
Yes, more specifically when the user does not have the necessary role to view the page.